setup.py

```diff
@@ -37,7 +37,7 @@ def load_requirements(*requirements_paths):
     requirements = set()
     for path in requirements_paths:
         requirements.update(
-            line.strip() for line in open(path).readlines()  # pylint: disable=consider-using-with
+            line.strip() for line in open(path, encoding="utf8").readlines()  # pylint: disable=consider-using-with
```

Comment:
pylint version [2.10.0](https://github.com/PyCQA/pylint/releases/tag/v2.10.0) deprecated the use of open() without explicitly specifying any encoding. See https://github.com/PyCQA/pylint/issues/3826 for details.